src/gui/element/catalog.rs

```diff
@@ -302,6 +313,8 @@ pub fn data_row_container<'a, 'b>(
         })
         .next()
     {
+        // TODO (casperstorm): localization timeago.
+        // @see: https://docs.rs/timeago/0.2.1/timeago/
```

Comment:
@tarkah This should properly also be localized, and they have support for it. If you have time you are very welcome to help.